### DISCUSSION TITLE
Use Code Analyzers only to the newly added code not the existing code

### DISCUSSION THREAD

**Discussion by sukreshmanda** [dotnet/roslyn#75216] [Q&A]
I have a very large legacy c# project, I want to include some custom code analysers to it. Since the project is very large and fixing them is not possible right now. I want to ensure the newly added code is follong those analyzers. Is there anyway to acheive this?

**Comment by CyrusNajmabadi** [comment]
Yes.  You can use editorconfig to mark the files/directories where you do it do not want analysis to happen.
